browser/brave_rewards/tip_dialog.cc

```diff
@@ -138,7 +144,10 @@ void OpenTipDialog(WebContents* initiator,
   content::WebContents* outermost_web_contents =
     guest_view::GuestViewBase::GetTopLevelWebContents(initiator);
   gfx::Size host_size = outermost_web_contents->GetContainerBounds().size();
-  const int width = host_size.width() - kDialogMargin;
+  int width = host_size.width() - kDialogMargin;
```

Comment:
Since we have some repetition here, can we make a method in this class that accepts a param (`gfx::Size`) and returns the calculated width?

Reply:
disregard - did not notice this was used in multiple namespaces